test/e2e/page-objects/PageBackend.js

```diff
@@ -8,13 +8,28 @@ const createTabAusstellungen = (t) => {
          */
         async checkAusstellungWirdAngezeigt(name) {
             const text = await Selector("#exhibition_table_body").innerText;
-            await t.expect(text).contains(name)
+            await t.expect(text).contains(name);
         },
-            /**
-             * Clicks the button to create a new exhibition (which will show a dialog)
-             * @returns Promise for the Page-Object of the 'Neue Ausstellung hinzufügen'-Dialog
-             */
-            async clickNeueAustellung() {
+        /**
+         * Check if the exhibition with the given name is not present on screen
+         * @param {*} name the name of the exhibition
+         */
+        async checkAusstellungWirdNichtAngezeigt(name) {
```

Comment:
This name (`checkAusstellungWirdNichtAngezeigt`) and the phpdoc of the method suggests that the function makes an explicit check whether the exhibition is visible or not. Indeed, it just verifies that no exhibition is visible. So either rename the function or scan the table if you cannot find an entry with the given name.